### ISSUE TITLE
Reproduction issues

### ISSUE THREAD

**Issue by ZGCTroy** [OliverRensu/xAR#10]
Thanks for your great work!

<img width="599" alt="Image" src="https://github.com/user-attachments/assets/626f67dd-14c7-40bf-9a53-97c0bf578991" />

The provided script uses "constant" lr scheduler and finds no support for cosine decay scheduler. I find learning at lr=4e-4, batch_size=2048 may lead to some convergence problems, so is cosine decay lr scheduler truly used in your experiment?

In addition, how many inference steps do you use in your paper, 40 or 50?

**Comment by ZGCTroy** [comment]
https://github.com/OliverRensu/xAR/blob/121e9c93df32295ebe79a52b4402a86a63f321f9/util/lr_sched.py#L4-L19

Actually, lr_scheduler='constant' is a fake hyperparameters.
